example/quoted-filename.hrx

```diff
@@ -4,5 +4,5 @@ Filenames may contain spaces as long as they're quoted.
 <===> "file\"name"
 Filenames may contain quotes as long as they're escaped.
 
-<===> file\"name
+<===> file\"name2
 Even unquoted filenames may contain escaped quotes.
```

Comment:
Can you just delete this whole file? We no longer treat quotes specially, those these test cases are irrelevant now.

Reply:
Alright, will do. I'll have to update my implementations for this that have special handling for quote characters. Should I also revert the escaped quote in the other file, or is that still important?